includes/module/model/CustomCheckout.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
```

Comment:
Colocar NOTICE OF LICENSE no header do arquivo (VIDE mercadopago.php)

Reply:
Toppen